### PR TITLE
Add Prometheus Config Reloader CPU and Memory flags

### PR DESCRIPTION
**Pull request by JessieAMorris**

### CODE REVIEW

**Comment by JessieAMorris** [comment]
This basically replicates https://github.com/coreos/prometheus-operator/pull/2403 but for the `prometheus-config-reloader` container. This enables us to resolve https://github.com/helm/charts/issues/11768

File: cmd/operator/main.go [hunk 1]
```diff
@@ -121,7 +121,9 @@ func init() {
 	// Prometheus Operator image, tagged with the same semver version. Default to
 	// the Prometheus Operator version if no Prometheus config reloader image is
 	// specified.
-	flagset.StringVar(&cfg.PrometheusConfigReloader, "prometheus-config-reloader", fmt.Sprintf("quay.io/coreos/prometheus-config-reloader:v%v", version.Version), "Prometheus config reloader image")
+	flagset.StringVar(&cfg.PrometheusConfigReloaderImage, "prometheus-config-reloader", fmt.Sprintf("quay.io/coreos/prometheus-config-reloader:v%v", version.Version), "Prometheus config reloader image")
+	flagset.StringVar(&cfg.PrometheusConfigReloaderCPU, "prometheus-config-reloader-cpu", "50m", "Prometheus config reloader CPU")
```
Comment (brancz) [inline cmd/operator/main.go]: given that these sidecars work almost identically, how about we just use the same flag for this?

Comment (JessieAMorris) [reply cmd/operator/main.go]: Same flag being just using the `config-reloader-cpu` and `config-reloader-memory` for both the `config-reloader` and `prometheus-config-reloader` containers?

Comment (brancz) [reply cmd/operator/main.go]: yes, that's what I was thinking

Comment (JessieAMorris) [reply cmd/operator/main.go]: I'd be good with that. I'll go ahead and change that.

**Comment by brancz** [comment]
lgtm :+1: thanks for the contribution!

**Pull request merged**